doc/rst/developer/bestPractices/TestSystem.rst

```diff
@@ -327,9 +508,24 @@ would use ``bar-100.perfkeys`` for its performance keys and write its
 output to ``bar-100.dat`` while the second would use ``bar-10000.perfkeys``
 and write its output to ``bar-10000.dat``.
 
+Comparing to a C version
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To compare a C version of a test to a Chapel version, the C version of
+the test must end with the suffix ``.test.c``.  Since ``.dat`` files must have
+unique names, the base name for the C test should vary from the Chapel
+equivalent.  For example, I might name the C version of the ``foo.chpl``
+performance test ``foo-c.test.c``.  Like any other test, the C test needs
+a ``.good`` file for correctness testing and a ``.perfkeys`` file for
+performance testing.
+
+C versions do not have to be performance tests, but this is their most common
+use case.
+
 
 Creating a graph comparing multiple variations
-----------------------------------------------
+++++++++++++++++++++++++++++++++++++++++++++++
+
```

Comment:
An existing line later on refers the reader to the performance graphs on sourceforge.  Is the chapel-lang site the official one now?